downstairs/src/main.rs

```diff
@@ -386,5 +425,64 @@ async fn main() -> Result<()> {
             );
             Ok(())
         }
+        Args::Dynamometer {
+            block_size,
+            data,
+            extent_size,
+            extent_count,
+            encrypted,
+            num_writes,
+            samples,
+            flush_per_iops,
+            flush_per_blocks,
+            flush_per_ms,
+        } => {
+            let uuid = Uuid::new_v4();
+
+            let region = create_region(
+                block_size,
+                data,
+                extent_size,
+                extent_count,
+                uuid,
+                encrypted,
+                log.clone(),
+            )
+            .await?;
+
+            eprintln!(
+                "{:?} {:?} {:?}",
+                flush_per_iops, flush_per_blocks, flush_per_ms
+            );
+
+            let mut flush_types_set = 0;
+            if flush_per_iops.is_some() {
+                flush_types_set += 1;
+            }
+            if flush_per_blocks.is_some() {
+                flush_types_set += 1;
+            }
+            if flush_per_ms.is_some() {
+                flush_types_set += 1;
+            }
+
+            if flush_types_set > 1 {
+                bail!("too many flush types set!");
+            }
+
+            let flush_config = if flush_per_iops.is_some() {
+                DynoFlushConfig::FlushPerIops(flush_per_iops.unwrap())
+            } else if flush_per_blocks.is_some() {
+                DynoFlushConfig::FlushPerBlocks(flush_per_iops.unwrap())
+            } else if flush_per_ms.is_some() {
+                DynoFlushConfig::FlushPerMs(flush_per_ms.unwrap())
+            } else {
+                panic!("wat");
```

Comment:
This is easily reachable by users:
```
cargo run -pcrucible-downstairs -- dynamometer --data tmp
    Finished dev [unoptimized + debuginfo] target(s) in 0.28s
     Running `target/debug/crucible-downstairs dynamometer --data tmp`
Sep 20 14:20:56.483 INFO raising number of open files limit to from 256 to 9223372036854775807
Sep 20 14:20:56.484 INFO Created new region file "tmp/region.json"
None None None
thread 'main' panicked at 'wat', downstairs/src/main.rs:480:17
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
fish: Job 1, 'cargo run -pcrucible-downstairs…' terminated by signal SIGABRT (Abort)
```

How about something like `bail!("must specify a flush interval");`?

Reply:
I added a `None` variant to test without flushing at all.